docs/data/date-pickers/localization/localization.md

```diff
@@ -77,7 +77,9 @@ If you want to pass language translations without using `createTheme` and `Theme
 you can directly load the language translations from the `@mui/x-date-pickers` or `@mui/x-date-pickers-pro` package and pass them to the `LocalizationProvider`.
 
 ```jsx
-import { LocalizationProvider, deDE, DatePicker } from '@mui/x-date-pickers';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { deDE } from '@mui/x-date-pickers/locales';
+import { DatePicker } from '@mui/x-date-pickers/DatePicker';
```

Comment:
For `@mui/material` I think that we do care because root import make dev build time a lot slower (since the dev build don't have tree shaking and `@mui/material` is huge).

Maybe that for our packages it's that that big of a deal indeed.